api/v1alpha1/nnfcontainerprofile_webhook.go

```diff
@@ -36,12 +36,18 @@ import (
 // log is for logging in this package.
 var nnfcontainerprofilelog = logf.Log.WithName("nnfcontainerprofile")
 
+// SetupWebhookWithManager will setup the manager to manage the webhooks
 func (r *NnfContainerProfile) SetupWebhookWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewWebhookManagedBy(mgr).
 		For(r).
 		Complete()
 }
 
+// TODO(user): EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+
+// TODO(user): change verbs to "verbs=create;update;delete" if you want to enable deletion validation.
+// NOTE: The 'path' attribute must follow a specific pattern and should not be modified directly here.
+// Modifying the path for an invalid path can cause API server errors; failing to locate the webhook.
```

Comment:
You want these TODOs in here?

Reply:
Per discussion today, I'm removing them.